### ISSUE TITLE
Files with no coverage

### ISSUE THREAD

**Issue by rkistner** [colszowka/simplecov#16]
Files without any coverage are not shown at all. It is important for me to be able to view these files.

As this is not detected by the built-in Coverage module, perhaps an option could be added to search for files in the current folder?

I'll try to create a proof-of-concept for this.

**Comment by adampope** [comment]
We encountered a problem with this method of tracking all files in #394 as it was marking lines as uncovered that shouldn't be.  So we went looking for another solution and we found that adding `Rails.application.eager_load!` after `SimpleCov.start` in `test_helper.rb` and making sure that our lib files were in eager_load_paths in `application.rb`

```
config.eager_load_paths += %W(#{config.root}/lib)
config.eager_load_paths += Dir["#{config.root}/lib/**/"]
```

solved all of our problems.  All files are being picked up for coverage purposes and we are not getting lines that should be ignored marked as uncovered.
